exporter/kafkaexporter/config.go

```diff
@@ -67,6 +68,13 @@ type Metadata struct {
 type Producer struct {
 	// Maximum message bytes the producer will accept to produce.
 	MaxMessageBytes int `mapstructure:"max_message_bytes"`
+
+	// RequiredAcks Number of Acknowledgements required to assume that a message has been sent.
```

Comment:
```suggestion
	// RequiredAcks Number of acknowledgements required to assume that a message has been sent.
```

Reply:
I would include this link here aswell https://pkg.go.dev/github.com/Shopify/sarama@v1.30.0#RequiredAcks